docs/src/main/asciidoc/microprofile-health.adoc

```diff
@@ -330,6 +330,16 @@ that will validate each datasources: link:datasource#datasource-health-check[Dat
 
 You can disable extension health check via the property `quarkus.health.extensions.enabled` so none will be automatically registered.
 
+== Health UI
```

Comment:
```suggestion
== Health UI (Experimental API not included in the MicroProfile specification yet)
```

Reply:
Thanks !

Reply:
I made it 
NOTE: Experimental - not included in the MicroProfile specification
so it's not part of the Heading.